### PR TITLE
Fix local studio logs

### PR DESCRIPTION
**Pull request by jordienr**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

- fix edge_logs and postgres logs queries.

### CODE REVIEW

File: apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts [hunk 1]
```diff
@@ -149,6 +162,16 @@ export const genDefaultQuery = (table: LogsTableName, filters: Filters) => {
   `
 
     case 'postgres_logs':
+      if (IS_PLATFORM === false) {
+        return `
+select id as identifier, postgres_logs.timestamp, id, event_message, parsed.error_severity
```
Comment (Ziinc) [inline apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts:167]:
```suggestion
select postgres_logs.timestamp, id, event_message, parsed.error_severity
```

id is not identifier, we can remove the `identifier field completely.

File: apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts [hunk 2]
```diff
@@ -138,8 +139,20 @@ export const genDefaultQuery = (table: LogsTableName, filters: Filters) => {
   const where = genWhereStatement(table, filters)
   const joins = genCrossJoinUnnests(table)
   const orderBy = 'order by timestamp desc'
+
   switch (table) {
     case 'edge_logs':
+      if (IS_PLATFORM === false) {
+        return `
+-- local dev edge_logs query
+select id, edge_logs.timestamp, event_message, metadata.event_type, metadata.function_id, metadata.level
```
Comment (Ziinc) [inline apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts:148]: why function_id and event_type?

File: apps/studio/components/interfaces/Settings/Logs/LogsQueryPanel.tsx [hunk 3]
```diff
@@ -99,25 +100,28 @@ const LogsQueryPanel = ({
...
-                    <DropdownMenuItem
-                      key={template.label}
-                      onClick={() => onSelectTemplate(template)}
-                    >
-                      <p>{template.label}</p>
-                    </DropdownMenuItem>
-                  ))}
-              </DropdownMenuContent>
-            </DropdownMenu>
+            {IS_PLATFORM && (
```
Comment (Ziinc) [inline apps/studio/components/interfaces/Settings/Logs/LogsQueryPanel.tsx:103]: might be good to have a `<PlatformOnly>` component for this kind of situations. just a thought

**Review by Ziinc — CHANGES_REQUESTED** [review]

File: apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts (hunk 2 above)
Comment (jordienr) [reply apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts:148]: was using another function as reference and i probably copypasted it 😅 nice catch thx

**Review by Ziinc — APPROVED** [review]

**Review by Ziinc — APPROVED** [review]

**Review by joshenlim — APPROVED** [review]

**Pull request merged**